vortex-duckdb/src/duckdb/data.rs

```diff
@@ -6,7 +6,7 @@ wrapper!(Data, cpp::duckdb_vx_data, |_| {});
 impl<T> From<Box<T>> for Data {
     fn from(value: Box<T>) -> Self {
         unsafe {
-            Self::own(cpp::duckdb_vx_data_create(
+            Self::borrow(cpp::duckdb_vx_data_create(
```

Comment:
But we Box::into_raw, so isn't this an owned Data?

Reply:
Its owned

Reply:
yes yes forgot to undo this one